test/e2e-common.sh

```diff
@@ -59,6 +59,101 @@ function install_pipeline_crd() {
   wait_until_pods_running tekton-pipelines || fail_test "Tekton Pipeline did not come up"
 }
 
+function spire_apply() {
```

Comment:
nit: add a comment here describing function and arguments

Reply:
added.